Adyen/adyen-magento2
# [ECP-8825] Enable pipeline runs for external contributions

**Description**

This change adds the possibility of running pipelines for PRs that are created by external contributors.

**Tested scenarios**


- Make sure the PRs by internal users still work as they should
- Make sure that the PRs from external contributors work properly, and they require a codeowner approval to start

Fixes  
ECP-8825
